scripts/effects/counterstance.lua

```diff
@@ -6,6 +6,10 @@
 local effectObject = {}
 
 effectObject.onEffectGain = function(target, effect)
+    if target:getFamily() == 59 then -- Bugbear Family
```

Comment:
This will print an error if a player uses counterstance
```suggestion
    if target:isMob() && target:getFamily() == 59 then -- Bugbear Family
```

Reply:
Fixed